src/css/application.css

```diff
@@ -315,6 +315,20 @@ body {
   border-bottom-left-radius: 0;
 }
 
+.top-bar__project-button-hidden {
+  visibility: hidden;
+}
+
+.top-bar__project-saved {
+  position: absolute;
+  visibility: hidden;
+  left: 1em;
+}
+
+.top-bar__project-saved-shown {
```

Comment:
Should be `top-bar__project-saved_shown` (single underscore delimits a modifier)